src/widgets/text.rs

```diff
@@ -69,9 +69,11 @@ impl TextWidget {
     }
 
     pub fn set_text(&mut self, content: String) {
-        if content.is_empty() {
-            self.spacing = Spacing::Hidden;
-        }
+        self.spacing = if content.is_empty() {
+            Spacing::Hidden
+        } else {
+            Spacing::Normal
+        };
```

Comment:
If we already do a check if text is empty, why not check if it's not empty?

It doesn't break any of the blocks I use.

Reply:
Not sure if it is something block author's would actually use, but the only thing I can think of is if a block sets the spacing to something other than `Normal` while still displaying text - this change would change the spacing behind the block's back.

Reply:
Without this change, this config
```toml
[[block]]
block = "net"
device = "..."
format = "{speed_down;K}{speed_up;K}"
format_alt = ""
```
after two clicks has incorrect spacing (no space on the right).

> change the spacing behind the block's back.

It already does it :eyes: